src/ankita/assignment/assignment14/ExplicitWait.java

```diff
@@ -0,0 +1,39 @@
+/*Form: Enable_And_Disable_Display_Hide_Form
+Script 1: check element's clickability using explicit wait
+Script 2: check element's visibility using explicit wait
+*/
+
+package ankita.assignment.assignment14;
+
+import ankita.base.PredefinedActions;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class ExplicitWait {
+	static WebDriver driver = PredefinedActions.start(
```

Comment:
you should call this method from any method, do not initialized this driver from the static variable.